sdk/core/azure_core/src/process/mod.rs

```diff
@@ -0,0 +1,51 @@
+use async_trait::async_trait;
+use std::io::Error;
+use std::{ffi::OsStr, process::Output};
+
+#[cfg(not(feature = "tokio_process"))]
+mod thread;
+
+#[cfg(not(feature = "tokio_process"))]
+pub use thread::ThreadRunner;
+
+#[cfg(feature = "tokio_process")]
+pub use tokio::TokioRunner;
+
+#[cfg(feature = "tokio_process")]
+mod tokio {
```

Comment:
It'd be better to just isolate these into separate modules and conditionally pull them in if enabled.

Reply:
Done